src/main/resources/templates/farmer/add.html

```diff
@@ -92,7 +99,7 @@ <h1 th:replace="fragments :: bannerFarmerLoggedIn"></h1>
     <td th:text="*{name}"></td>
     <td th:text="*{productCategory}"></td>
     <td th:text="'$'+*{ProductDetails.price}"></td>
-    <td>Placeholder</td>
+    <td th:text="*{ProductDetails.quantity}"></td>
```

Comment:
I think this should be:

 <td th:text="*{ProductDetails.quantity+' '+measurementcategory.name}"></td>

Reply:
` <td th:text="*{ProductDetails.quantity+' '+measurementcategory.name}"></td>`